linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_maximum_age_login_defs/tests/correct_cis.fail.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I would rename this scenario to something like `incorrect_cis.fail.sh`.

Reply:
Thanks for catching @matusmarhefka. Copy and past without attention is a poison. : ))

Reply:
Done